packages/expo-file-system/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### 🎉 New features
 
+- Add path joining in `File` and `Directory` constructors. ([#31467](https://github.com/expo/expo/pull/31467) by [@aleqsio](https://github.com/aleqsio))
```

Comment:
It would be worth prefixing changelog entries with `[next]` or something similar to indicate that they are part of the new API. What do you think?

Reply:
```suggestion
- [next] Add path joining in `File` and `Directory` constructors. ([#31467](https://github.com/expo/expo/pull/31467) by [@aleqsio](https://github.com/aleqsio))
```